view/frontend/layout/checkout_onepage_success.xml

```diff
@@ -2,8 +2,10 @@
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
         <referenceContainer name="order.success.additional.info">
-            <block class="Omise\Payment\Block\Checkout\Onepage\Success\TescoAdditionalInformation" name="onepage.success.billing_agreement" 
-            template="Omise_Payment::checkout/onepage/success/tesco_additional_info.phtml"/>
+            <block class="Omise\Payment\Block\Checkout\Onepage\Success\TescoAdditionalInformation" name="onepage.success.tesco" 
```

Comment:
Just wonder if renaming one to another won't affect anything to the previous version of plugin?
Does this require merchant to clear store's caches or something?

Reply:
no threat. 
it was mistake in the past, and it was the same name as paypal.
That was probably bringing problems.
We need to change it, but we could fix it in separated PR if that is what would be better here. @guzzilar

Reply:
🙆‍♂